src/backend/distributed/commands/extension.c

```diff
@@ -491,6 +495,137 @@ PreprocessAlterExtensionUpdateStmt(Node *node, const char *queryString)
 }
 
 
+/*
+ * PostprocessAlterExtensionCitusUpdateStmt is called after ALTER EXTENSION
+ * citus UPDATE command is executed by standard utility hook.
+ *
+ * Actually, we do not need such a post process function for ALTER EXTENSION
+ * UPDATE commands unless the extension is Citus itself. This is because we
+ * need to mark existing objects that are not included in distributed object
+ * infrastructure in older versions of Citus (but now should be) as distributed
+ * if we really updated Citus to the available version successfully via standard
+ * utility hook.
+ */
+List *
+PostprocessAlterExtensionCitusUpdateStmt(Node *node, bool
+										 citusCanBeUpdatedToAvailableVersion)
+{
+	if (!CitusHasBeenLoaded())
```

Comment:
This check is probably redundant, since we wouldn't have gotten here if the Citus extension did not exist.

Reply:
I think so, standard utility hook would already error out if Citus is not installed